pkg/apis/core/v1alpha1/service_bindings.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+type ServiceBinding struct {
+	Name      string
+	SecretRef *corev1.LocalObjectReference
```

Comment:
Does this need to be a pointer?

Reply:
probably not, but the binding in the provisioned service is nilable, so a pointer made sense